owncloudApp/src/main/res/layout/share_public_dialog.xml

```diff
@@ -11,7 +11,7 @@
         android:layout_height="wrap_content"
         android:padding="@dimen/standard_half_padding"
         android:text="@string/share_via_link_create_title"
-        android:textAppearance="@android:style/TextAppearance.DeviceDefault.DialogWindowTitle" />
+        android:textAppearance="@style/Theme.ownCloud.AlertDialog.WindowTitle" />
```

Comment:
I just checked and an easier option is applying:

`android:textAppearance="@android:style/TextAppearance.Material.DialogWindowTitle"`

It implies just 1 change and the looking is similar to the current one (but correct color). The new style you created makes the title a bit smaller 😄